importer_config.template.py

```diff
@@ -0,0 +1,129 @@
+from os import path
```

Comment:
So, I'd go for a different and more readable structure. Try this - a 2d config. 

config['BOTANY']['SPECIFY_DATABASE'] = "casbotany"
config['ICH']['RANDOM_param'] = "foo"
etc
etc
then when you get_config you specify the section and you get back a 1d array of just those items. Then front load the whole thing so we see the defs up top, and not a bunch of code, which will be at the bottom.

e.g.:

get_config('BOTANY')

def get_config(section_name):
  if section name not in config
    explode 
  return config[section_name)